Editors/ImportExportEditor/Editors.ImportExport/DevConfig/Import_Karl.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.IO;
+using Editors.ImportExport.Exporting.Exporters.RmvToGltf;
+using Editors.ImportExport.Importing.Importers.GltfToRmv;
+using Editors.ImportExport.Importing.Importers.GltfToRmv.Helper;
+using Shared.Core.DevConfig;
+using Shared.Core.PackFiles;
+using Shared.Core.PackFiles.Models;
+using Shared.Core.Services;
+using Shared.GameFormats.RigidModel;
+using Shared.GameFormats;
+using SharpGLTF.Schema2;
+using Shared.GameFormats.RigidModel.LodHeader;
+using System.Windows.Forms.Design;
+using Shared.Core.Settings;
+
+namespace Editors.ImportExport.DevConfig
+{
+    public class Import_Karl : IDeveloperConfiguration
+    {
+        private readonly IPackFileService _packFileService;
+        private readonly GltfImporter _importer;
+
+        public Import_Karl(IPackFileService packFileService, GltfImporter exporter)
+        {
+            _packFileService = packFileService;
+            _importer = exporter;
+        }
+
+        public void OpenFileOnLoad()
+        {
+            const string inputGltfAe = @"C:\Users\Kaiza\Documents\Maharaja_GLTF_WeightCrap\PhazerMade\dae_prince_torso_base_01.gltf";
```

Comment:
Update the paths to be relative and part of the repo. 
if this is more of a one off thing, dont add a devConfig for it